ipn/ipnlocal/local.go

```diff
@@ -2901,33 +2901,70 @@ func (b *LocalBackend) CheckIPForwarding() error {
 	}
 	if isBSD(runtime.GOOS) {
 		return fmt.Errorf("Subnet routing and exit nodes only work with additional manual configuration on %v, and is not currently officially supported.", runtime.GOOS)
+	} else if runtime.GOOS != "linux" {
```

Comment:
I missed this at first. Maybe invert this to `runtime.GOOS == "linux"` and make it `return b.checkIPForwardingLinux()` instead, pulling the rest out into a Linux-specific func? (in same file is fine for now)